rootconf/default/lib/onie/onie-uninstaller-common

```diff
@@ -0,0 +1,174 @@
+# common uninstall routine
+
+#  Copyright (C) 2014,2015 Curt Brune <curt@cumulusnetworks.com>
+#  Copyright (C) 2016 david_yang <david_yang@accton.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# for backward compatibility if running partition_type is empty assume it
+# is msdos.
+true ${onie_partition_type="msdos"}
```

Comment:
In powerpc platforms, `onie_partition_type` is empty.  And most platforms do not have GPT tools.  So that it should be msdos by default.